argo-event/templates/sensor.yaml

```diff
@@ -0,0 +1,40 @@
+apiVersion: argoproj.io/v1alpha1
+kind: Sensor
+metadata:
+  name: {{ .Values.name }}
+  {{- with .Values.labels }}
+  labels:
+    {{ toYaml . }}
+  {{- end }}
+spec:
+  eventBusName: {{ .Values.name }}
+  {{- with .Values.eventsensor.config }}
+  replicas: {{ .replicas | default 1 }}
+  {{- end }}
+  template: 
+  {{- with .Values.annotations }}
+    annotations:
+    {{ toYaml . }}
+  {{- end }}
+  dependencies:
+    {{- range $element := .Values.eventsensor.deplendencies }}
+      - name: {{ $element.dependencyName }}
+        eventSourceName: {{ $.Values.name }}
+        eventName: {{ $element.eventName }}
+        {{- if $element.filter }}
+        filter:
+          {{ toYaml $element.filter }}
+        {{- end }}
+    {{- end }}
+  triggers:
+    {{- range $element := .Values.eventsensor.triggers }}
+      - template:
+          name: {{ $element.name }}
+          {{- if eq $element.type "http" }}
+          {{- template "trigger_http" $element }}
+          {{- end }}
+          log:
```

Comment:
What's the usage of this log field

Reply:
to display log for this trigger

ref: https://github.com/argoproj/argo-events/blob/master/api/sensor.md#logtrigger

Reply:
1. Have you tested it in your env? The syntax is a little bit weird as you are creating a multi-trigger sensor (http + log) here, but the syntax should look like this AFAIK: https://github.com/argoproj/argo-events/blob/master/examples/sensors/multi-trigger-sensor.yaml
2. The log trigger should only be used for debugging so I don't think it should be enabled by default https://argoproj.github.io/argo-events/sensors/triggers/log/, you may add support for the user to explicitly creating log trigger
3. If your goal is to enable debug log for a HTTP trigger, perhaps you should try this instead? https://github.com/argoproj/argo-events/blob/master/examples/sensors/log-debug.yaml